.flake8

```diff
@@ -0,0 +1,20 @@
+[flake8]
```

Comment:
Can we remove this and continue to pass things as cli args? I'm happy to accept a config file, but i'd rather not change too much at once!

Reply:
f183f13